packages/performance/src/services/remote_config_service.test.ts

```diff
@@ -37,7 +37,6 @@ describe('Performance Monitoring > remote_config_service', () => {
   "fpr_log_endpoint_url":"https://firebaselogging.test.com",\
   "fpr_log_transport_key":"pseudo-transport-key",\
   "fpr_log_source":"2","fpr_vc_network_request_sampling_rate":"0.250000",\
-  "fpr_log_transport_web_percent":"100.0",\
```

Comment:
Can we keep one test which preserves `fpr_log_transport_web_percent` field in the config? Therefore we can validate that Web SDK doesn't fail while removing the definition of this flag.

Reply:
Good idea. Have added a test to validate that parsing of other relevant parameters are not impacted.